### ISSUE TITLE
Publish on F-Droid

### ISSUE THREAD

**Issue by alexanderadam** [inbasic/turbo-download-manager#32]
I would be happy if the android build would also be [available in F-Droid market](https://f-droid.org/repository/browse/?fdfilter=turbo+download+manager&fdpage=1&page_id=0).

Thank you!

**Comment by inbasic** [comment]
Yes, why not! Though I am not really familiar with F-Droid market. What are the requirements? Would you be willing to take care of the submission?

**Comment by alexanderadam** [comment]
I don't really understand how apps are published on F-Droid.

To be honest it looks a bit unstructured for me as there is just a kind of [submission queue forum](https://f-droid.org/forums/forum/submission-queue/) (without a visible search field) where people people post, more or less, unstructured metadata as well.
But I guess things like name, description, build instructions ([ie. will the instruction in the readme be enough for someone with android building background?](https://github.com/inbasic/turbo-download-manager/tree/a336ff01853b3f09cb5f1928ea901521e12f445e#technical-notes)), repository and license are relevant. Which also brings me to the point of requirements. It must be [free & open source software](https://f-droid.org/wiki/page/Inclusion_Policy) but I can't find a `LICENSE` file in the repository? I don't know whether [declaring which license](https://github.com/inbasic/turbo-download-manager/blob/a336ff01853b3f09cb5f1928ea901521e12f445e/src/package-electron.json#L21) is enough or a copy of the license must be provided.

Addionally it seems as there aren't any git release tags in this repositoy(?) which [would be ideal for proper releases](https://f-droid.org/wiki/page/Inclusion_Policy).

That said I could create the forum entry on the submission queue but when it comes to technical questions I, most probably, couldn't answer these.

PS: If we are writing anyway: **Thank you for your work so far!**

**Comment by inbasic** [comment]
I posted on F-Droid submission queue to find if [CCA based](https://github.com/MobileChromeApps/mobile-chrome-apps) projects are supported.
https://f-droid.org/forums/topic/support-status-for-chrome-cordova-apps-cca/

If we got positive response, I'll try to fulfill the requirements

**Comment by alexanderadam** [comment]
You are really awesome! :+1:
